common/perception_utils/include/perception_utils/object_classification.hpp

```diff
@@ -92,6 +93,81 @@ inline bool isLargeVehicle(const std::vector<ObjectClassification> & object_clas
   auto highest_prob_label = getHighestProbLabel(object_classifications);
   return isLargeVehicle(highest_prob_label);
 }
+
+inline uint8_t toLabel(const std::string & class_name)
+{
+  if (class_name == "UNKNOWN") {
+    return ObjectClassification::UNKNOWN;
+  } else if (class_name == "CAR") {
+    return ObjectClassification::CAR;
+  } else if (class_name == "TRUCK") {
+    return ObjectClassification::TRUCK;
+  } else if (class_name == "BUS") {
+    return ObjectClassification::BUS;
+  } else if (class_name == "TRAILER") {
+    return ObjectClassification::TRAILER;
+  } else if (class_name == "MOTORCYCLE") {
+    return ObjectClassification::MOTORCYCLE;
+  } else if (class_name == "BICYCLE") {
+    return ObjectClassification::BICYCLE;
+  } else if (class_name == "PEDESTRIAN") {
+    return ObjectClassification::PEDESTRIAN;
+  } else {
+    return ObjectClassification::UNKNOWN;
+  }
+}
+
+inline ObjectClassification toObjectClassification(
+  const std::string & class_name, float probability)
+{
+  ObjectClassification classification;
+  classification.label = toLabel(class_name);
+  classification.probability = probability;
+  return classification;
+}
+
+inline std::vector<ObjectClassification> toObjectClassifications(
+  const std::string & class_name, float probability)
+{
+  std::vector<ObjectClassification> classifications;
+  classifications.push_back(toObjectClassification(class_name, probability));
+  return classifications;
+}
+
+inline std::string toString(const uint8_t label)
```

Comment:
[imo] I feel this function name is ambiguous (used like `perception_utils::toString` from other packages), since other perception_msgs also have the constant values, e,g, [autoware_auto_perception_msgs/msg/TrafficLight](https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/TrafficLight.idl) , [autoware_auto_perception_msgs/msg/Shape](https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/Shape.idl)

Reply:
fixed at [802b254](https://github.com/autowarefoundation/autoware.universe/pull/2090/commits/802b254e4f8dae67a3c84009127d4073c6e04088)
(Do you have any suggestion for alternative function name?)

Reply:
sorry I don't have any idea about a better function name. Another idea is to create a nested namespace like `perception_utils::classification::toString`

Reply:
If other function is used as `perception_utils::isVehicle()`, I think it is intuitive and better to align for namespace like `perception_utils::convertLabelToString()`.